engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -391,19 +392,25 @@ class MetadataBuilderActorSpec
     class TestReadDatabaseMetadataWorkerActorForCost extends ReadDatabaseMetadataWorkerActor(defaultTimeout, 1000000) {
       override def receive: Receive = {
         case GetCost(wfId) if wfId == mainWorkflowId =>
-          sender() ! CostResponse(mainWorkflowId, workflowState, MetadataLookupResponse(mainQuery, mainEvents))
+          sender() ! CostResponse(mainWorkflowId, workflowRunningState, MetadataLookupResponse(mainQuery, mainEvents))
           ()
         case GetCost(wfId) if wfId == subWorkflow1Id =>
-          sender() ! CostResponse(subWorkflow1Id, workflowState, MetadataLookupResponse(sub1Query, sub1Events))
+          sender() ! CostResponse(subWorkflow1Id, workflowSucceededState, MetadataLookupResponse(sub1Query, sub1Events))
           ()
         case GetCost(wfId) if wfId == subWorkflow2Id =>
-          sender() ! CostResponse(subWorkflow2Id, workflowState, MetadataLookupResponse(sub2Query, sub2Events))
+          sender() ! CostResponse(subWorkflow2Id, workflowSucceededState, MetadataLookupResponse(sub2Query, sub2Events))
           ()
         case GetCost(wfId) if wfId == subWorkflow1aId =>
-          sender() ! CostResponse(subWorkflow1aId, workflowState, MetadataLookupResponse(sub1aQuery, sub1aEvents))
+          sender() ! CostResponse(subWorkflow1aId,
```

Comment:
Nothing to do really and naming things are hard anyways, but I had to reread this 3 times before I noticed the 1, vs 1a, vs 1b, vs 2 😓